linuxdoc/manKernelDoc.py

```diff
@@ -55,7 +55,6 @@ def setup(app):
 
     app.add_builder(KernelDocManBuilder)
     app.add_directive("kernel-doc-man", KernelDocMan)
-    app.add_config_value('author', "", 'env')
```

Comment:
@GabrielGanne FYI this value is needed in sphinx version < 1.8 / see commit d004d3e

Reply:
You're right. My mistake.

I believe this could have been fixed without even testing sphinx version by:
``` python
if 'author' not in app.config:
    app.add_config_value('author', "", 'env')
```

Reply:
@GabrielGanne right, but IMO it also has a value to implement compatibility issues along sphinx versions (which is the responsible).  Anyway, if you think your solution is better, send a PR and I will accept it.